### ISSUE TITLE
enter stage as guest

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/mobilise#258]
when i am entering a stage as a guest, the log-in fields are much more dominant than the "enter as guest" button. it should be the other way around - as players will more often be entering the stage from Backstage (formerly known as the dashboard) and be already logged in.
also, as guest, i don't get an invitation to enter a username. could this screen be changed so that it assumes that you are a guest, and has a field for you to enter a username before you enter; and then underneath all of that, there is a "player login" button which would then reveal the fields for you to log in.

![Screenshot at 2021-04-10 16-12-23](https://user-images.githubusercontent.com/3437324/114272726-9898ad00-9a17-11eb-9d64-879a1af6800a.png)

**Comment by vickis** [comment]
yes agree :)

**Comment by elisadcg** [comment]
Ok so we need an intermediat step 
![34-logginguest](https://user-images.githubusercontent.com/7611913/114675913-0f2ff600-9d09-11eb-9045-dddb5dd36758.png)

**Comment by nguyenhongphat0** [comment]
What is the different between `Login as an Audience` and `Login as a Player`, Elisa 👀 From what I see it will lead you to the same login form, so why don't we merge these 2 button into 1

**Comment by elisadcg** [comment]
True but they don't know that it's the same login form. so maybe we can join the tow button in Login (audience or player)

**Comment by helenvarley** [comment]
no - audience should NOT be asked to log in - this is confusing. 

after they click "enter as a guest" - and in fact, it would be better to say "enter as audience" - there should be a window that says "Would you like to have a nickname?" and a field where they can enter a nickname, but also the option to enter without a nickname (leaving the field blank)

they are not logging in, and we should not be gathering any data from them.

**Comment by vickis** [comment]
Action should be
Enter as audience (no login)
chat name field is presented OR
once in the stage can change name in chat

Or login - as player

**Comment by nguyenhongphat0** [comment]
So in conclusion we will have two form: `audience` and `player`. Audience form will be the default 👌 

<img width="45%" alt="Screenshot 2021-05-06 at 15 23 02" src="https://user-images.githubusercontent.com/29925961/117265944-f5547f80-ae7e-11eb-9440-c79826abf287.png"> <img width="45%" alt="Screenshot 2021-05-06 at 15 23 29" src="https://user-images.githubusercontent.com/29925961/117266021-056c5f00-ae7f-11eb-816a-b510fd35edbc.png">

**Comment by vickis** [comment]
great!

**Comment by helenvarley** [comment]
yes, great! one thing: can there be an option for audience to enter without chosing a nickname? maybe it's enough to have a line of text: "Choose a nickname if you want one. Click the green button to enter the stage."
or maybe that's obvious?

**Comment by elisadcg** [comment]
i don't think it's obvious

**Comment by nguyenhongphat0** [comment]
Yes they can choose their nickname later. They can hit the green button or click on the background to skip

**Comment by helenvarley** [comment]
How about adding this: 
"To enter the stage, click on the green button. You can choose a nickname if you want one."

**Comment by nguyenhongphat0** [comment]
It's quite long but I will add it for you 👌

**Comment by helenvarley** [comment]
maybe "To enter the stage, click the green button." is enough?
we can let people work it out themselves that they can enter a nickname or not ... ??

**Comment by vickis** [comment]
Could Choose a nickname be in grey in the  text box (as a prompt which
disappears as you start typing)

And enter text be on Green button

No nickname is also a choice
:)

**Comment by nguyenhongphat0** [comment]
> Enter text on the green button

It look quite weird to me 😅 I can bounce the green button to make it stand out if you want to

<img width="1310" alt="Screenshot 2021-05-07 at 16 22 44" src="https://user-images.githubusercontent.com/29925961/117428648-76c81280-af50-11eb-985d-f952555f7419.png">
